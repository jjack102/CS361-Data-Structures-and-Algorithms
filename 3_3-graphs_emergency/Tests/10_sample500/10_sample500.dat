0 499 0
0 276 2 0
276 150 7 0
150 237 20 0
237 216 11 0
216 461 6 0
461 252 17 0
252 245 1 1
245 431 19 0
431 224 6 1
224 376 7 1
376 113 19 0
113 35 19 0
35 428 18 0
428 234 16 0
234 361 4 0
361 0 1 1
0 104 17 0
104 90 16 1
90 434 12 0
434 31 8 0
31 11 7 1
11 463 20 0
463 67 13 0
67 81 10 0
81 499 15 1
258 310 15 1
485 72 16 1
142 465 3 1
338 60 17 0
63 240 5 0
473 108 8 1
399 109 9 1
384 48 3 0
59 407 5 0
197 452 13 0
381 415 16 0
299 331 16 0
37 453 10 1
105 107 17 0
302 55 19 0
474 60 15 1
218 312 15 0
133 212 5 1
277 323 19 1
272 340 6 0
416 97 5 0
301 315 19 1
58 37 2 0
215 118 12 0
444 413 20 0
422 326 9 0
328 221 6 1
9 217 13 0
26 114 5 1
324 328 14 0
170 299 11 0
389 240 7 1
111 374 14 0
452 388 13 1
49 224 16 0
80 465 7 0
173 233 16 0
281 232 16 1
271 348 11 0
350 67 14 0
261 124 14 0
245 441 8 0
197 254 18 0
158 223 14 0
221 483 1 0
52 243 16 0
282 293 10 0
483 393 20 0
19 435 5 0
133 68 8 0
159 65 14 0
34 332 3 1
299 167 11 0
416 422 17 1
367 266 8 0
489 129 14 1
242 235 4 1
70 439 19 1
341 229 5 0
15 62 15 1
64 67 14 0
305 346 7 0
212 288 16 0
91 231 19 0
223 76 2 0
299 234 3 0
141 185 11 1
419 292 16 0
204 335 6 0
66 72 6 1
454 56 14 0
53 196 5 1
143 475 4 0
488 484 9 0
56 3 19 0
399 120 4 0
422 204 17 0
466 486 7 0
345 409 20 1
217 68 15 1
485 368 14 1
155 479 20 1
54 74 13 0
123 110 10 0
261 190 18 0
34 215 14 0
246 87 18 0
264 241 11 0
415 104 12 0
77 471 1 0
107 476 14 1
92 381 14 0
33 410 19 0
367 378 5 0
320 84 9 0
298 157 3 0
304 453 7 0
136 210 18 0
379 260 8 0
321 141 20 1
450 355 6 1
208 48 8 0
295 472 2 0
289 210 16 0
76 366 1 1
33 35 15 0
195 125 13 0
268 228 8 0
36 135 12 1
106 208 20 0
71 404 3 0
365 376 5 0
358 354 15 0
86 89 20 0
114 259 13 0
230 416 19 0
467 439 16 0
464 114 1 0
263 105 7 0
176 192 3 0
323 209 4 1
227 18 14 0
335 348 9 1
246 223 18 0
483 204 5 0
283 407 3 0
109 148 3 0
135 319 9 0
354 221 13 0
408 468 11 0
333 491 3 1
236 300 15 0
282 52 18 0
266 45 8 1
9 442 15 0
89 333 12 0
151 317 10 1
203 338 6 0
17 302 9 0
413 0 1 0
6 196 16 0
259 467 5 0
197 394 2 1
138 51 12 0
468 47 4 0
372 301 11 0
390 108 3 0
453 172 9 0
13 219 4 0
69 22 16 0
161 159 9 0
355 164 8 0
328 411 15 1
3 345 20 0
213 268 11 0
197 219 9 0
465 457 15 1
39 162 9 0
351 177 9 0
65 319 1 0
224 437 13 0
472 466 2 0
398 425 9 0
490 411 7 0
496 60 9 0
433 157 2 1
71 141 1 1
174 399 11 0
52 29 5 0
96 265 10 0
297 293 1 0
336 262 1 0
250 375 14 1
276 454 2 0
53 342 20 0
125 129 18 1
279 98 6 0
96 177 19 0
375 86 13 1
88 372 9 1
498 374 11 0
381 69 2 0
15 55 10 0
320 67 9 0
84 416 13 0
184 382 9 1
62 20 20 1
496 164 20 0
396 273 2 1
118 301 7 1
115 244 7 0
430 246 17 0
194 378 7 0
276 69 9 0
317 216 16 0
441 167 11 0
90 323 7 0
237 457 20 1
230 409 15 1
24 315 2 0
326 60 19 0
49 388 12 0
81 45 15 0
52 329 3 0
318 23 3 0
130 285 20 0
119 412 17 1
32 313 13 0
143 308 16 0
234 460 13 0
376 176 11 0
127 467 1 1
351 216 7 0
424 433 17 0
126 301 20 0
248 304 12 0
361 297 12 0
44 294 8 0
343 448 13 0
123 13 13 0
471 105 3 1
299 442 9 0
370 331 13 1
255 13 14 0
491 490 6 0
325 367 2 0
18 304 2 0
186 221 5 0
21 463 14 1
154 499 13 1
264 255 11 1
155 477 3 0
52 199 19 0
124 247 20 0
147 99 3 1
261 471 11 0
205 60 9 0
413 427 17 1
243 21 3 0
440 86 12 1
363 117 8 1
305 233 4 1
421 492 19 0
306 294 1 0
307 460 1 1
293 423 8 0
67 448 9 0
400 472 5 0
499 376 3 1
219 499 3 0
476 332 9 1
322 33 15 1
244 235 15 0
460 339 9 1
108 252 4 0
425 323 9 0
369 231 16 1
56 290 4 0
75 214 4 0
85 382 1 0
27 230 9 1
157 94 7 1
31 57 10 0
66 382 16 1
286 138 15 0
381 262 10 0
322 203 14 0
295 67 11 0
291 255 6 0
406 251 17 1
7 184 10 1
75 376 4 0
267 483 9 0
18 342 15 1
198 455 11 0
76 225 8 0
38 233 4 0
319 421 8 1
142 238 17 0
303 428 5 0
376 37 5 0
190 198 18 0
240 337 9 0
117 383 6 0
92 376 19 0
135 99 7 0
60 210 10 0
45 198 9 0
153 449 9 0
468 191 15 0
34 140 19 0
112 176 10 0
378 416 20 1
457 254 6 0
453 499 4 0
304 45 2 0
337 274 8 0
150 416 17 1
233 210 11 0
478 60 14 1
389 23 7 0
258 262 12 1
267 264 14 1
211 87 19 0
455 245 4 1
227 420 5 0
273 105 18 0
188 82 15 1
211 493 7 0
82 16 16 0
390 271 14 0
142 171 9 0
52 292 8 0
87 285 2 0
354 65 11 0
198 243 10 0
441 308 19 0
5 208 6 0
72 68 2 0
295 380 9 1
313 375 20 0
300 438 18 1
480 340 4 0
21 192 6 0
333 455 7 1
60 223 12 1
78 202 18 0
395 482 14 0
494 334 9 0
268 200 15 0
252 318 1 1
443 193 9 0
478 300 6 0
253 51 6 0
368 400 9 0
283 33 15 0
275 435 20 0
471 256 16 0
275 344 19 1
300 108 12 1
151 360 2 0
183 229 17 0
414 430 7 0
212 260 8 0
265 332 16 0
460 17 6 1
150 255 11 0
62 480 11 0
55 315 19 1
31 275 6 0
215 166 11 0
308 491 6 0
101 131 3 0
99 376 4 1
328 399 14 0
336 479 17 0
111 20 2 0
61 377 10 0
302 273 15 1
489 67 14 0
105 449 3 0
436 107 11 1
56 498 5 1
87 463 12 0
165 471 9 1
238 381 2 0
137 398 8 0
224 239 12 0
381 175 13 0
467 209 14 0
426 304 14 0
41 389 17 0
176 102 4 0
266 478 6 0
375 255 14 0
398 332 20 1
56 186 2 0
291 464 5 0
37 230 15 1
427 202 10 0
201 165 5 0
83 191 6 1
415 159 10 1
282 296 18 0
334 249 4 0
358 260 12 1
405 399 2 0
149 81 15 1
353 267 5 0
412 228 6 0
374 162 14 0
412 282 10 0
200 203 11 0
246 237 2 1
433 173 11 0
440 487 12 0
190 375 5 1
178 457 3 0
35 460 5 1
254 221 20 0
61 464 4 0
482 467 8 1
215 483 10 0
400 68 13 0
103 164 5 0
296 477 4 0
260 27 7 1
263 6 6 0
116 196 19 0
32 56 14 1
183 222 1 1
432 213 8 1
180 467 13 0
173 286 3 0
40 87 2 0
102 374 9 1
54 317 5 1
358 454 13 0
385 313 10 1
471 33 12 0
290 149 7 0
93 402 17 1
28 208 12 0
261 145 2 0
404 293 20 0
201 440 11 0
367 133 12 0
5 399 8 1
452 31 11 0
37 493 6 1
118 139 17 0
367 172 9 1
308 77 2 0
478 193 19 0
390 449 18 0
78 33 10 0
99 171 9 0
287 127 9 0
379 48 7 0
272 136 10 0
193 326 17 0
428 413 9 1
311 285 9 0
164 264 9 0
316 202 2 0
109 434 3 0
193 328 19 0
458 2 11 1
258 437 17 1
47 0 16 0
238 478 3 1
33 189 10 0
149 173 10 0
169 381 3 0
221 293 19 0
369 267 17 0
119 149 3 1
180 431 8 0
434 196 4 0
47 251 14 0
250 8 1 0
37 444 15 0
336 496 3 1
298 94 3 0
285 197 10 0
95 375 5 0
175 480 2 0
65 177 7 0
5 234 15 0
410 227 2 1
27 131 13 0
346 499 20 0
280 378 14 0
430 66 10 0
74 329 14 0
169 325 7 0
337 207 3 1
121 173 5 1
55 271 10 0
120 192 4 0
468 67 5 0
67 496 14 0
399 132 17 1
173 405 13 0
218 435 10 0
484 246 7 0
66 182 4 0
97 198 9 1
60 379 17 0
116 134 10 0
208 286 13 0
311 45 16 0
345 432 1 0
70 400 20 0
203 42 19 0
92 24 12 0
461 159 6 1
20 361 12 0
144 45 5 0
118 362 16 0
332 454 18 0
460 250 20 1
136 57 12 0
448 196 1 1
436 416 6 0
182 496 11 0
202 494 18 0
393 47 5 0
278 316 18 0
299 334 19 0
358 243 10 0
210 320 18 0
255 230 15 1
33 399 8 0
127 151 5 0
113 311 17 0
457 426 17 0
61 93 6 0
483 361 15 0
289 272 8 1
120 480 8 0
169 14 3 0
379 448 4 0
330 31 15 1
134 40 3 0
126 344 8 0
360 381 15 1
426 245 19 1
132 405 17 0
248 98 12 1
216 205 1 0
24 277 14 1
298 359 7 0
138 439 16 1
131 415 17 0
263 252 13 0
247 432 13 0
118 479 1 0
232 448 14 0
362 218 20 0
62 201 1 0
370 340 19 0
446 354 3 1
245 195 8 0
452 461 7 0
252 476 12 0
238 51 2 0
336 376 7 0
490 319 16 0
144 105 20 0
423 243 17 1
390 41 8 0
222 474 11 0
489 84 17 0
193 330 15 1
73 107 3 0
31 295 15 0
448 477 14 0
149 193 14 1
24 454 9 0
155 277 15 0
282 393 18 0
180 118 9 0
122 22 15 1
290 266 6 0
480 213 8 0
361 370 10 0
255 84 15 1
344 244 15 0
20 389 20 1
426 445 12 0
497 457 8 0
240 445 8 1
287 242 15 1
138 311 17 0
196 145 13 0
440 478 7 0
39 472 6 0
449 13 14 0
495 239 11 0
131 475 17 0
452 492 4 0
197 59 5 0
76 393 11 0
156 449 14 0
283 82 4 1
394 280 4 0
40 135 2 0
77 327 7 1
229 67 16 0
490 277 3 0
213 283 13 1
255 104 9 0
255 56 8 0
117 102 1 0
295 100 12 1
429 100 8 0
92 478 7 1
11 21 18 1
372 167 7 0
470 155 19 0
250 364 7 0
287 142 4 0
321 432 9 0
135 163 8 0
209 348 12 0
446 465 3 0
304 54 3 0
373 273 5 0
8 21 5 1
373 290 8 0
121 466 6 0
268 484 15 0
339 140 3 0
152 309 1 0
147 254 6 0
25 286 11 0
396 229 12 0
354 69 4 0
161 490 4 0
232 371 15 0
338 31 13 1
188 495 11 0
85 61 12 0
268 445 17 0
82 494 4 0
235 55 3 1
312 4 3 0
40 151 1 1
496 192 9 1
312 496 6 0
298 189 11 1
282 194 18 0
418 137 3 1
263 432 6 0
479 347 16 0
303 317 13 0
378 343 1 0
164 315 7 0
404 433 3 0
261 339 7 0
279 348 19 1
394 91 7 1
352 434 6 0
242 201 10 0
478 406 19 0
197 276 13 1
95 331 3 0
322 366 2 1
320 437 7 0
298 299 3 0
285 453 19 0
469 163 1 0
296 133 14 0
479 200 10 0
418 240 3 0
39 197 14 1
88 286 8 0
288 293 6 0
220 382 18 0
346 51 12 0
288 43 13 0
327 236 16 0
226 2 17 1
102 47 4 0
439 400 16 0
346 224 16 0
353 315 5 1
388 149 16 0
301 367 16 1
201 71 6 0
459 93 4 0
269 399 10 0
379 409 2 0
192 451 16 0
144 38 13 0
2 432 6 0
433 182 14 1
168 160 5 0
36 122 1 0
207 475 17 0
22 405 18 1
200 227 13 0
73 440 1 0
228 374 11 0
159 282 4 0
445 118 5 0
375 66 10 0
17 106 10 0
328 62 13 1
57 472 9 1
452 412 16 0
404 386 14 1
446 425 18 0
46 334 11 0
399 105 10 0
309 274 6 0
10 361 12 0
1 435 12 0
153 82 6 0
161 312 19 0
364 459 2 0
282 91 20 0
25 152 8 1
49 353 14 0
214 458 18 0
325 166 17 1
370 82 20 0
404 168 13 1
359 302 12 0
354 110 15 1
407 164 15 1
384 270 11 0
377 238 11 0
205 31 9 0
320 367 7 0
195 36 9 0
326 478 9 1
479 203 11 0
130 28 1 0
409 196 8 0
486 86 9 0
214 209 14 0
168 119 17 0
229 379 17 0
421 84 14 0
490 181 4 0
248 226 15 0
451 477 13 1
464 8 7 1
8 136 5 0
227 204 19 1
24 53 13 0
34 3 5 0
109 164 14 0
31 18 2 1
212 18 7 0
104 426 8 1
79 125 10 1
397 308 19 0
356 171 16 0
392 198 1 0
352 492 19 0
425 155 5 0
470 241 12 0
163 330 15 0
378 243 17 1
34 402 20 1
148 420 11 1
406 257 8 0
84 289 2 1
275 148 17 0
159 232 11 0
75 238 20 1
357 472 20 0
47 65 18 0
143 439 1 0
264 347 15 0
284 41 5 0
2 254 5 0
282 18 2 0
84 12 3 0
415 261 11 0
391 173 3 1
19 328 10 0
314 146 17 0
476 474 10 1
378 403 3 1
212 87 20 0
228 111 1 0
153 223 4 0
403 269 15 0
71 187 20 1
310 425 19 0
293 92 11 0
295 377 11 1
457 408 2 1
200 372 14 0
170 444 10 0
397 189 17 0
124 211 9 1
335 100 14 0
37 66 1 0
356 102 11 0
5 84 2 0
213 158 19 1
307 468 12 1
279 230 4 0
155 89 17 0
156 300 6 0
34 451 5 0
30 491 12 0
360 230 12 1
215 30 13 0
26 464 5 1
71 150 13 1
175 258 17 0
251 65 3 1
176 459 14 1
167 182 17 0
43 232 7 0
192 202 3 0
201 472 4 0
433 208 1 0
413 441 20 0
9 447 7 0
244 39 14 0
290 104 13 0
269 5 7 1
486 296 7 0
469 409 6 0
298 497 3 0
168 49 16 0
62 344 11 1
360 81 2 0
26 403 7 1
313 71 5 1
458 366 7 0
43 243 19 0
75 456 20 0
184 436 5 0
256 428 5 0
475 46 8 1
385 96 16 0
52 223 13 0
244 373 4 0
133 43 12 0
370 153 4 0
444 284 15 0
497 305 20 1
217 376 12 0
60 383 5 0
447 18 19 0
249 342 8 1
261 176 1 0
298 297 18 0
112 54 4 0
78 87 16 0
101 315 10 0
36 5 16 0
38 280 4 0
378 23 4 0
323 101 17 0
176 120 17 0
237 26 20 0
425 455 11 0
402 485 14 0
338 201 10 0
4 439 9 1
43 117 4 0
116 193 15 1
267 228 19 0
248 345 14 0
168 201 13 0
12 204 16 1
58 402 16 0
336 288 19 0
426 160 9 0
389 454 17 1
280 127 10 0
480 57 1 0
434 234 10 0
42 272 7 0
435 398 1 0
211 330 20 0
368 179 16 0
24 135 16 0
408 124 9 1
332 428 11 1
177 344 15 0
132 235 1 0
246 320 8 0
23 172 13 0
127 464 6 1
244 459 13 0
21 178 17 1
194 64 7 0
302 481 10 1
314 13 1 0
312 182 7 1
45 188 17 0
317 453 19 0
312 149 20 0
233 341 12 0
345 217 19 0
76 92 8 0
37 99 7 0
116 370 3 0
216 95 15 0
334 460 20 0
55 356 7 0
138 101 11 1
272 440 5 0
82 86 5 0
306 246 13 0
121 351 18 1
286 438 9 0
156 450 12 1
440 389 14 0
291 285 11 0
106 367 12 0
229 143 15 0
319 346 17 1
365 35 1 0
293 200 17 0
496 200 10 0
56 134 7 1
301 180 7 0
427 236 12 0
118 85 1 1
482 239 5 0
436 269 17 0
178 444 9 1
219 470 14 0
333 363 15 0
107 291 17 0
230 337 13 0
286 401 9 0
35 152 20 1
437 328 5 0
352 285 9 0
29 260 10 0
271 330 6 0
292 198 2 0
66 410 18 0
283 401 6 0
22 180 15 0
15 93 8 0
150 348 14 1
456 257 16 1
491 39 4 1
94 278 7 0
440 129 4 0
430 229 1 0
310 134 3 1
14 339 11 0
246 286 11 0
21 38 2 0
336 88 8 0
448 472 5 1
341 98 17 0
43 215 11 0
278 411 16 0
308 280 6 1
111 117 18 1
38 103 17 1
156 132 13 0
381 96 13 0
114 163 1 1
326 424 11 0
297 192 18 1
115 395 18 0
330 136 4 0
433 167 3 0
224 381 14 0
139 417 20 0
132 462 1 0
445 293 12 0
243 57 8 1
410 281 5 1
160 66 10 1
265 393 10 0
14 231 8 0
408 192 10 0
7 57 14 0
385 122 3 0
452 215 12 1
259 385 10 0
234 335 11 1
118 373 10 0
253 302 4 1
260 237 17 0
265 205 10 1
30 8 20 1
114 440 9 0
