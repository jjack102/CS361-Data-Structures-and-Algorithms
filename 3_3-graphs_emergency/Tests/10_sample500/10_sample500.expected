Fastest time for car at 0 to the emergency at 499 is 41 minutes.
